src/client/app/actions/conversions.ts

```diff
@@ -99,7 +99,7 @@ export function submitEditedConversion(editedConversion: t.ConversionData, shoul
 				showSuccessNotification(translate('conversion.successfully.edited.conversion'));
 			} catch (err) {
 				// Failure! ):
-				showErrorNotification(translate('conversion.failed.to.edit.conversion'));
+				showErrorNotification(translate('conversion.failed.to.edit.conversion') + '"' + err + '"');
```

Comment:
In other places in OED we do "err.response.data" to see the actual message. I think it may be a little risky if another type of error but I think we should do it here and in the other messages to be consistent.